packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -30,9 +30,30 @@ const assertAggregationIsDefinedForCodesInFormula = (
   });
 };
 
+const assertAllDefaultsAreCodesInFormula = (
+  defaultValues: DefaultValuesSpecs = {},
+  { formula }: { formula: ArithmeticConfig['formula'] },
+) => {
+  const variables = getVariables(formula);
+  Object.keys(defaultValues).forEach(code => {
+    if (!variables.includes(code)) {
+      throw new Error(`'${code}' is in defaultValues but not referenced in the formula`);
+    }
+  });
+};
+
+const assertAllValuesAreNumbers = (defaultValues: DefaultValuesSpecs = {}) => {
```

Comment:
```suggestion
const assertAllDefaultValuesAreNumbers = (defaultValues: DefaultValuesSpecs = {}) => {
```

Reply:
I have gone with a different approach to keep that validator general. I would appreciate it if you could re-review 🙏

Reply:
Good on you for being proactive! I would suggest a similar change 😆

Reply:
😆, perhaps you are finally rubbing off on me.